packages/transport-ble/src/transports/BleInboundTransport.ts

```diff
@@ -0,0 +1,82 @@
+import type { Central } from '@animo-id/react-native-ble-didcomm'
+import type { Agent, InboundTransport, Logger } from '@aries-framework/core'
+import type { EmitterSubscription } from 'react-native'
+
+import { TransportService, utils, MessageReceiver } from '@aries-framework/core'
+
+import { BleTransportSession } from './BleTransportSession'
+
+export class BleInboundTransport implements InboundTransport {
+  public supportedSchemes: string[] = ['ble']
+  private central: Central
+  private messageListener?: EmitterSubscription
+  private session!: BleTransportSession
+  private disconnectionListener?: EmitterSubscription
+  private logger!: Logger
+
+  public constructor(central: Central) {
+    this.central = central
+  }
+
+  public async start(agent: Agent): Promise<void> {
+    this.logger = agent.config.logger
+
+    this.logger.debug('Starting BLE inbound transport')
+
+    const sessionId = utils.uuid()
+
+    this.session = new BleTransportSession(sessionId, this.central, agent)
+
+    const messageListener = async (data: unknown) => {
```

Comment:
Yes, we can include this in the SDK. But until then, we can't define the type in the callback because the typing on the listener is still `string`, which causes it to throw an error if it's not either `string` or `unknown`.

Reply:
I've updated this in the SDK and made a PR: https://github.com/animo/react-native-ble-didcomm/pull/33